File: Year02/2nd semester/MPP/Lab/mpp-proiect-java-AlexandruPaul21/JavaProjectNetworking/ProjectNetworking/src/main/java/org/example/dto/AgencyDto.java
package org.example.dto;

import java.io.Serializable;

public class AgencyDto implements Serializable {
    private String username;
    private String name;
    private String password;

    public AgencyDto(String username, String name, String password) {
        this.username = username;
        this.name = name;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    @Override
    public String toString() {
        return "AgencyDto{" +
                "username='" + username + '\'' +
                ", name='" + name + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}

File: Year02/2nd semester/MPP/Lab/mpp-proiect-csharp-AlexandruPaul21/NetworkingProject/networking/AgencyDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace networking
{
    [Serializable]
    public class AgencyDto
    {
        public String username { get; set; }
        public String name { get; set; }
        public String password { get; set; }

        public AgencyDto(String username, String name, String password)
        {
            this.username = username;
            this.name = name;
            this.password = password;
        }
    }
}
